shared/chat/audio/audio-recorder.native.tsx

```diff
@@ -30,9 +30,7 @@ const AudioRecorder = (props: Props) => {
   const ampTracker = React.useRef(new AmpTracker()).current
   const [visible, setVisible] = React.useState(false)
   const [closingDown, setClosingDown] = React.useState(false)
-  const {audioRecording} = Container.useSelector(state => ({
```

Comment:
this makes a new object everytime, for no reason, so it thrashes on every store change